internal/cmd/init_test.go

```diff
@@ -0,0 +1,7 @@
+package cmd
```

Comment:
Wouldn't this be better suited in `internal/test/util` or perhaps `internal/test`? I would also rename this file to `init.go` when moved there.

Reply:
@tjasko so this means that in all files that require these schemas, I need to ensure to import that test/util module? am I right? (if not already used)

Reply:
note that this is the main one: https://github.com/Kong/koko/pull/444/files#diff-7bc2adc8698fa2d6855fd6659a309ec9294ab00c02699e11b20f06862d46a689R184

this file intend is only to ensure that function is ran before running tests.

Reply:
Ah, we spoke about this on the side, but you can mind me. I wasn't fully thinking this through, and for whatever reason didn't notice some other changes being added.